apps/backoffice-v2/src/domains/storage/fetchers.ts

```diff
@@ -39,10 +39,12 @@ export const fetchFileInfoById = async (fileId: string) => {
 };
 export const fetchFileById = async (fileId: string) => {
   const fileInfo = await fetchFileInfoById(fileId);
+
```

Comment:
I would appreciate if you could comment and say in the PR that this is a boyscouting thing. It will make the code review go smoother. 🙏